### ISSUE TITLE
Screws up other plugins

### ISSUE THREAD

**Issue by edge0703** [formulahendry/vscode-auto-close-tag#31]
This plugin seems to have a major bug. Whenever I use Alt + Cmd + . to close a tag, all other plugins are screwed up and don't work anymore. This also goes for the native tooltip function in VS Code (that shows information about HTML tags), it also stops working after that.

**Comment by formulahendry** [comment]
Could you please list the impacted extension lists?

**Comment by edge0703** [comment]
It's mostly https://marketplace.visualstudio.com/items?itemName=letrieu.expand-region. But like said, the native tag helper tooltips also don't work anymore after I tried to close a tag.

**Comment by formulahendry** [comment]
If you disable the expand-region, and try again, will native tag helper tooltips work?

**Comment by edge0703** [comment]
Nope, expand-region doesn't seem to have to do anything with this bug. Disabled it, tried to close a tag with Alt + Cmd + ., and suddenly the tag helper tooltips didn't work anymore.

**Comment by formulahendry** [comment]
Tooltips like below? It works normally in my machine
![close-tag](https://cloud.githubusercontent.com/assets/1050213/23801038/89350924-05e9-11e7-9328-d323b618976e.gif)

**Comment by edge0703** [comment]
Yes, those. Then it must have to do with another constellation of plugins. Will try to disable others and see if that helps - and will get back to you then. Thanks for your help.
